dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/Helper.java

```diff
@@ -713,30 +289,11 @@ public static Token extractToken(List<Token> tokens, String source) {
     }
 
     /**
-     * Gets containers for the current user
-     *
-     * @param userId
-     * @param userDAO
-     * @return
-     */
-    private static List<Tool> getContainers(Long userId, UserDAO userDAO) {
-        final Set<Entry> entries = userDAO.findById(userId).getEntries();
-        List<Tool> toolList = new ArrayList<>();
-        for (Entry entry : entries) {
-            if (entry instanceof Tool) {
-                toolList.add((Tool)entry);
-            }
-        }
-
-        return toolList;
-    }
-
-    /**
      * @param reference
      *            a raw reference from git like "refs/heads/master"
      * @return the last segment like master
      */
-    private static String parseReference(String reference) {
+    public static String parseReference(String reference) {
```

Comment:
This might be a candidate to move to QuayImageRegistry if it is only used there. Let's try to gradually shrink Helper, moving its method into more descriptive classes.